firefox-ios/Client/Experiments/Messaging/GleanPlumbMessage.swift

```diff
@@ -82,6 +83,13 @@ struct GleanPlumbMessage {
     var surface: MessageSurfaceId {
         data.surface
     }
+
+    /// The survey options for this message if it has a microsurvey configuration.
+    ///
+    /// Embedding apps should not read from this directly.
+    var options: [String] {
+        data.microsurveyConfig?.options ?? []
```

Comment:
add `return` here

Reply:
@adudenamedruby there are other computed variables without `return`, I will update that in a separate PR. thanks!